service/src/tests/mocks/parentchain_api_mock.rs

```diff
@@ -0,0 +1,72 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use itp_api_client_extensions::{ApiResult, ChainApi, StorageProof};
+use itp_test::builders::{
+	parentchain_block_builder::ParentchainBlockBuilder,
+	parentchain_header_builder::ParentchainHeaderBuilder,
+};
+use itp_types::{SignedBlock, H256};
+use sp_finality_grandpa::AuthorityList;
+use std::cmp::min;
+
+pub struct ParentchainApiMock {
```

Comment:
Short comment here would be nice.

Reply:
Thanks @mullefel for your effort - I dare to assume you've wanted to represent my preferences here 💯  But I had to concede a little bit about amassing comments (because comments are often left out when the code is updated). I think case I think the struct name should be enough.

@gaudenzkessler Thanks for adding a comment, but I don't think this comment does help in any way. If we have a comment here I'd prefer something that actually adds information.

Reply:
I can agree to that, yes